website/docs/ssh/README.md

```diff
@@ -53,13 +53,47 @@ on your computer that you'd like to use.
 
 #### 9. When prompted, press Enter twice to use a blank passphrase
 
-#### 10. Use the following command to display your public SSH key
+#### 10. Display your public SSH key
+
+<Tabs>
+<TabItem value="linux" label="Linux" default>
 
 ```bash
 cat ~/.ssh/id_rsa.pub # Copy it with Shift + Ctrl + C
 ```
 
-#### 11. Copy your SSH key to the fleet manager website (`New SSH Key`)
+</TabItem>
+
+<TabItem value="windows" label="Windows" default>
+
+a. Open the command prompt (Windows Key + R -> cmd -> Enter)
+
+b. On the terminal window type:
+
+```bash
+cd .ssh
+start .
+```
+
+c. Hit **Enter**
+
+d. This will open the folder where your `SSH` key was saved.
+
+e. Open the `id_rsa.pub` on a text editor such as Notepad.
+
+f. Copy the entire content of the file (including ssh-rsa) and paste in the website (Step 7)
```

Comment:
But a better option would be to combine the two sections (display & copy) as a single section (copy)
```suggestion
f. Copy the entire content of the file (including ssh-rsa) and paste in the website (Step 11)
```